### PR TITLE
Modify electra loss calcualation part

### PR DESCRIPTION
**Pull request by BM-K**
`loss_fct(logits.view(-1, self.num_labels), labels.view(-1))` and `loss_fct(logits, labels)` do the same thing, the latter code is more efficient.

### CODE REVIEW

**Review by sgugger — COMMENTED** [review]
It's not equivalent. Your suggested changes will error if the `labels` have a shape of [1, num_labels]  while the logits have a shape of [hidden_size, num_labels].
